wca-regulations.md

```diff
@@ -463,7 +463,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - B2a) The judge resets the timer(s) as in [Regulation A3b](regulations:regulation:A3b) and indicates that they are ready for the competitor to start the attempt (e.g. placing the puzzle cover in front of the competitor, saying "READY", giving a thumbs-up signal). The competitor must start the attempt within one minute, else the competitor forfeits the attempt (DNS), at the discretion of the judge.
     - B2b) When using the Stackmat timer, the competitor uses their fingers to touch the elevated sensor surfaces of the timer. The competitor's palms must be facing down, and located on the side of the timer that is closer to them. Penalty: time penalty (+2 seconds).
     - B2c) When using the Stackmat timer, the competitor must have no physical contact with the puzzle or the puzzle cover while starting the attempt. Penalty: time penalty (+2 seconds).
-    - B2d) When using the Stackmat timer, the competitor starts the attempt by removing their hands from the timer, thus starting the timer. (This also starts the solve.)
+    - B2d) When using the Stackmat timer, the competitor starts the attempt by removing their hands from the timer and starting the timer. (This also starts the solve.)
```

Comment:
```suggestion
    - B2d) When using the Stackmat timer, the competitor should keep their hands on the timer until they see a green timer light. The timer is started when they remove their hand(s) from the timer. The competitor starts the attempt and the solve by starting the timer.
```
Not sure if we should update "When using the Stackmat timer" to "If a Stackmat timer is in use" since that is what is used in Article A